src/globalStyle.js

```diff
@@ -0,0 +1,11 @@
+import { createGlobalStyle } from "styled-components";
+
+const GlobalStyle = createGlobalStyle`
+    * {
+        margin: 0;
+        padding: 0;
+        box-sizing: border-box;
+    }
+`;
+
+export default GlobalStyle;
```

Comment:
es una práctica común dejar una nueva línea al final del documento, aquí un link con algunas de las ventajas de hacerlo https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Es bueno saberlo, gracias! Nueva línea agregada al final de todos los archivos a lo que les faltaba en el nuevo commit a52aa80